### DISCUSSION TITLE
Viewing list of watchers/forks/stars

### DISCUSSION THREAD

**Discussion by OlivierJM** [community/community#19528] [Other Feature Feedback, Questions, & Ideas] [answered]
It used to be easy to click on number of forks on my personal repo and see a list of people who forked that repo and their respective forks, same used to apply for watchers and stars but now everytime I attempt to do this it seems to only trigger the actions for the main button.   

Is this the intended behaviour? Or Am I doing something wrong? I tried navigating with the keyboard too and it seems to be the same issue.   
Below the image, I have used the screenshot from this repo. 

<img width="449" alt="Screenshot 2022-06-27 at 8 12 13 AM" src="https://user-images.githubusercontent.com/11255454/175871582-764932ac-9479-49aa-984a-f76fe1fa97fd.png">

**Comment by melvincwng** [comment]
@OlivierJM Hi Oli, I believe the functionality of those buttons may have been changed.

Just FYI, if you want to check the number of watchers, stargazers, forks of a particular repo, you can add the following to the end of your repo's URL:

1. /stargazers
2. /watchers
3. /fork

This will reveal the people who starred, watched and forked your repository respectively.

If this answer has been helpful, don't forget to 'Mark as answer'.

Thank you and have a nice day ahead 😄

**Comment by OlivierJM** [comment, marked as answer]
Months later, I just found out from this similar discussion that the quick link is elsewhere not where it used to be https://github.com/orgs/community/discussions/8720
